docs/docs/cloud/cloud-roadmap.mdx

```diff
@@ -1,6 +1,7 @@
 ---
-id: early-access
+id: cloud-roadmap
```

Comment:
If this is updated we need a redirect in ory/web too!

Reply:
I will prepare a draft PR for the redirect, but I see no reason to keep this, right?

Reply:
Yes, no reason to keep this :)

Reply:
But redirect is needed, pretty sure we have it linked somewhere!

Reply:
https://github.com/ory/web/pull/421